recipe/meta.yaml

```diff
@@ -8,14 +8,22 @@ source:
   fn: fftw-{{ version }}.tar.gz
   url: http://www.fftw.org/fftw-{{ version }}-pl1.tar.gz
   sha256: 1ef4aa8427d9785839bc767f3eb6a84fcb5e9a37c31ed77a04e7e047519a183d
+  # This patch should be applied to the installed fftw3.h, not at build time
+  # patches:
+  #   - fix-dynamic.patch  # [win]
 
 build:
-  skip: True  # [win]
   number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py >= 35]
 
 requirements:
   build:
     - perl 5.22.2.1
+    - cmake >=3  # [win]
+    - toolchain  # [win]
 
 test:
```

Comment:
Need to add...

```yaml
test:
  requires:
    - python 2.7.*  # [win and py27]
    - python 3.4.*  # [win and py34]
    - python 3.5.*  # [win and py35]
    - python 3.6.*  # [win and py36]
```

This is necessary to make sure the feature is selected correctly for testing.